portal-api/src/v1/users/transform-database-user.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-param-reassign */
```

Comment:
yay

Reply:
I think there will be lots of other places this can be done now I've update the linting service-wide. That said I dontthink its part of this PR

Reply:
Brilliant, would be great to create a ticket for this

Reply:
https://ukef-dtfs.atlassian.net/browse/DTFS2-7603 done